response_operations_ui/views/accounts.py

```diff
@@ -40,6 +49,58 @@ def get_my_account():
         return jsonify(e)
 
 
+@account_bp.route("/my-account", methods=["POST"])
+def update_account():
+    form = OptionsForm()
+    form_valid = form.validate()
+    if not form_valid:
+        flash("You need to choose an option")
+        return redirect(url_for("account_bp.get_my_account"))
+    else:
+        return redirect(url_for(form_redirect_mapper.get(form.data["option"])))
+
+
+@account_bp.route("/change-account-name", methods=["GET", "POST"])
+def change_account_name():
+    try:
+        form = ChangeAccountName()
+        user_id = session["user_id"]
+        user_from_uaa = uaa_controller.get_user_by_id(user_id)
+        user = {
+            "first_name": f"{user_from_uaa['name']['givenName']}",
+            "last_name": f"{user_from_uaa['name']['familyName']}",
+        }
+        payload = user_from_uaa
```

Comment:
No need for this line.  Just use `user_from_uaa` in the lines below this, or call it something else that you'd be happy using everywhere